src/components/Books.jsx

```diff
@@ -1,15 +1,28 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 // Component imports
 import Book from './Book';
 import AddBookForm from './AddBookForm';
+import { getData } from '../redux/books/booksSlice';
```

Comment:
@KayLemba In this case, if it is disabled it will still give a warning. Even though the warning is there it can't be followed or the program loops infinitely. 
![image](https://user-images.githubusercontent.com/114885671/214756118-2398faaf-fa0c-4788-b2d0-da1082392328.png)
That is the warning. I re-enabled the linters but the warning will not go away.
Here is the full view of the code that causes the warning.
![image](https://user-images.githubusercontent.com/114885671/214756236-5c6edc23-0892-493d-a929-cc3768c6dee1.png)